package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pushwoosh-cordova-plugin",
-  "version": "6.5.3-OS4",
+  "version": "7.8.3",
```

Comment:
Shouldn't the version also be "something-OS"?

Reply:
Or does the upgrade "remove" our customizations?

Reply:
Yes, should be, we remove almost customizations made except one callback. I was thinking to define that on a separated commit to raise the plugin version.

Reply:
I agree, if we have changed a single line of code we should add the -OS suffix.